sst.py

```diff
@@ -2316,7 +2317,7 @@ def destroybase():
                 pdest.charted = True
             game.probe.moves -= 1 # One less to travel
             if game.probe.arrived() and game.isarmed and pdest.stars:
-                supernova(game.probe)                # fire in the hole!
+                supernova(game.probe.quadrant())                # fire in the hole!
```

Comment:
Please add an informative log msg
to commit. Ideally it should contain
the python's crash message.
Like:
```
fix crash because of missing quadrant() call

This fixes the following crash:
<crash msg here>
```